organization/_posts/2024/2024-03-28-coordination.md

```diff
@@ -0,0 +1,140 @@
+- A stronger, better defined role for the *Core Coordination Group*
+- A new *Advisory Group*, where we formally gather experiment feedback
+- New ideas on what an *HSF Project* means: this can include *HSF Reference Implementations* and some badge for acknowledging a project's work
+    - There will be minimum guidelines on software quality
+- A role the HSF could play as a *Virtual Centre of Excellence for Scientific Software*
+    - This would be alongside other, funded, activities
+
+## Working Group Updates
+
+### Data Analysis
+
+- Workflow language workshop next week, co-organized by Jamie (https://indico.cern.ch/event/1380367/). Want to use event to gauge interest from speakers in contributing to a dedicated DAWG talk series on workflow languages.
+
+### PyHEP
+
+- [PyHEP 2024](https://indico.cern.ch/event/1384010/) (1–4 July, 2024) registration and call for abstracts is open now
+- [PyHEP.dev 2024](https://indico.cern.ch/e/PyHEP2024.dev) (26–30 August, 2024) organisation continues
+
+### Detector Simulation
+
+- Geant4 Technical forum to take place on 11th of April: <https://indico.cern.ch/event/1394848/>
+
+### Software Training
+
+- WLCG-HSF Workshop:
+    - Parallel session:
+        - Invited talks
+        - Time for discussions - How to interact to generate a bigger effect in terms of training
+
+- Pre-CHEP organization:
+    - Pre-CHEP weekend before CHEP
+
+- Next Training Events:
+    - May 20-21, Software Carpentry
+    - May 27 scikit-hep/ROOT
+
+#### C++ Course and Hands-on Training
+
+- 10th HEP C++ Course and Hands-on Training - The Essentials, Mar 25 -29 <https://indico.cern.ch/event/1370681/>
+    - It's running right now!
+
+### Reconstruction and Software Trigger
+
+- Organising a topical meeting on "4D tracking @ Belle II":
+    - Confirmed date: 8th May
+    - Agenda: <https://indico.cern.ch/event/1399441/>
+    - Not announced in mailing list yet.
+    - Two talks: "Hit Timing Determination in Belle2 SVD" (Luigi Corona) & "SVD Timing in Tracking at Belle2" (Thomas Lueck)
+    - Thanks to Giulia for providing the speakers and Joe for following up.
+
+### JuliaHEP
+
+- Topical group meetings:
+    - April, May and June
+- JuliaHEP 2024 Workshop at CERN (end of September - beginning of October):
+    - JuliaHEP Training before the workshop
+
+### Tools and Packaging
+
+- The conveners will organise a chat soon.
+
+## Other Interest and Activity Areas
+
+### Analysis Facilities
+
+The Analysis Facilities Forum [Whitepaper](https://docs.google.com/document/d/1Pn9KWG-tGQ20OaNFUVlXLQddC7vFsQnu2EHR4DBfTjo/edit?usp=sharing) is revised and finalised. Many people have added their endorsement. If you still want to then please [email the AFF conveners](mailto:hsf-af-forum-convenors@googlegroups.com) *right away*.
+
+### Compute and Accelerator Forum
+
+[Indico agenda](https://indico.cern.ch/category/12741/)
+
+- April: high-performance I/O for HEP
+- May: GPU infrastructure and tools being developed; CERN infrastructure update
+
```

Comment:
The June AMD event is confirmed for the GPU roadmap. Another AMD event for CPU is planned (but not confirmed yet) for after summer.

Reply:
Thanks - it's done!